### ISSUE TITLE
openFAST v3.5.0 crashing when setting BldNd_BladesOut to 0 in ED.dat

### ISSUE THREAD

**Issue by TimoHuelsmann** [OpenFAST/openfast#1651]
**Bug description**
<A clear and concise description of the bug.>
The bug occurred when running openFAST v3.5.0 with the IEA15MW monopile model. 
To reduce the total number of output channels, the blade node outputs were tried to be eliminated in the ElastoDyn.dat, by setting the BldNd_BladesOut to 0 (as hinted in the readthedocs manual). 
The simulation aborts immediately, stating the error  _forrtl: severe (157): Program Exception - access violation_ in the console.

The error did not occur, when also removing the OutList-entries associated with the node output setting.

**To Reproduce**
<Update the following list with your specific information.>

To achieve this error, the ElastoDyn.dat section
```
---------------------- NODE OUTPUTS --------------------------------------------
 1   BldNd_BladesOut  - Blades to output
 99   BldNd_BlOutNd   - Blade nodes on each blade (currently unused)
              OutList     - The next line(s) contains a list of output parameters.  See OutListParameters.xlsx, ElastoDyn_Nodes tab for a listing of available output channels, (-)
"TDx"    - local flapwise (translational) deflection (relative to the undeflected position) of node
"TDy"    - local edgewise (translational) deflection (relative to the undeflected position) of node
"RDz"
END of input file (the word "END" must appear in the first 3 columns of this last OutList line)
---------------------------------------------------------------------------------------
```
has been changed to 
```
---------------------- NODE OUTPUTS --------------------------------------------
0   BldNd_BladesOut  - Blades to output
99   BldNd_BlOutNd   - Blade nodes on each blade (currently unused)
              OutList     - The next line(s) contains a list of output parameters.  See OutListParameters.xlsx, ElastoDyn_Nodes tab for a listing of available output channels, (-)
"TDx"    - local flapwise (translational) deflection (relative to the undeflected position) of node
"TDy"    - local edgewise (translational) deflection (relative to the undeflected position) of node
"RDz"
END of input file (the word "END" must appear in the first 3 columns of this last OutList line)
---------------------------------------------------------------------------------------
```
. The error occurred when running the simulation from windows power shell, as well as when running the simulation with Simulink.

**Expected behavior**
<A clear and concise description of what you expected to happen.>
I would have expected that it is enough to set the BldNd_BladesOut  to  0 to remove the blade node outputs from the .out-files.

It is still possible to successfully run openFAST and remove the output channels when also removing all OutList entries assoicuiated with the node output (e.g. removing "TDx", "TDy" and "RDz")  but I feel, that should not be necessary. 

**OpenFAST Version**
<Please provide as much detail as possible including git commit. The best information is a screenshot of the OpenFAST system description that prints when running OpenFAST:>

```
**************************************************************************************************
OpenFAST

Copyright (C) 2023 National Renewable Energy Laboratory
Copyright (C) 2023 Envision Energy USA LTD

This program is licensed under Apache License Version 2.0 and comes with ABSOLUTELY NO WARRANTY.
See the "LICENSE" file distributed with this software for details.
**************************************************************************************************

OpenFAST-v3.5.0
Compile Info:
 - Compiler: Intel(R) Fortran Compiler 2021
 - Architecture: 64 bit
 - Precision: single
 - OpenMP: No
 - Date: May 17 2023
 - Time: 13:43:33
Execution Info:
 - Date: 06/26/2023
 - Time: 11:15:54+0200

OpenFAST input file heading:
    IEA 15 MW offshore reference model monopile configuration

forrtl: severe (157): Program Exception - access violation
Image              PC                Routine            Line        Source
openfast_x64.exe   00007FF68666F9F0  Unknown               Unknown  Unknown
openfast_x64.exe   00007FF6865FA4E2  Unknown               Unknown  Unknown
openfast_x64.exe   00007FF6865A717B  Unknown               Unknown  Unknown
openfast_x64.exe   00007FF685BE9839  Unknown               Unknown  Unknown
openfast_x64.exe   00007FF685BFC967  Unknown               Unknown  Unknown
openfast_x64.exe   00007FF684D7034A  Unknown               Unknown  Unknown
openfast_x64.exe   00007FF684D8B9EE  Unknown               Unknown  Unknown
openfast_x64.exe   00007FF684CF1294  Unknown               Unknown  Unknown
openfast_x64.exe   00007FF6875EC22E  Unknown               Unknown  Unknown
openfast_x64.exe   00007FF687B15110  Unknown               Unknown  Unknown
KERNEL32.DLL       00007FFBDB327614  Unknown               Unknown  Unknown
ntdll.dll          00007FFBDC6226F1  Unknown               Unknown  Unknown
```
**System Information:**
 - OS: Microsoft Windows 10 Home v10.0.19045 Build 19045 

**Additional context**
<Add any other context about the problem here.>
When running the simulation with the above settings using Matlab-Simulink, Matlab crashes.

**Comment by andrew-platt** [comment]
This is definitely a bug. I'll take a look at it and get a fix for v3.5.1.

Another way you can remove the node outputs is to simply delete the `NODE OUTPUTS` section of the input file (this is the only section that this works for).  But I agree, this should not be necessary to do.

**Comment by andrew-platt** [comment]
Fixed with #1654
